DotaStats/PlayersModule/PlayersModule.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+class PlayersModule {
+    private let view: PlayersModuleInput
+    private let presenter: PlayersModuleOutput
```

Comment:
Лучше снаружи не знать, что у вас внутри есть какой-либо `presenter`
```suggestion
    let output: PlayersModuleOutput
```

Еще часто может понадобиться, чтобы наружу торчал `input` вашего модуля.